File: Photon Tools/Servers/Game/python/scenes/Map_1/Portals/Object 1/Portal.py
from MathematicsHelper import *
from InterestManagement import *
from Game.Application.GameObjects import *
from Game.Common import Maps

import os
import json

path = os.path.abspath(__file__)
jsonPath = os.path.splitext(path)[0] + ".json"

with open(jsonPath) as data_file:
    data = json.load(data_file)

x = data["Position"]["x"]
y = data["Position"]["y"]

position = Vector2(x, y)
destinationMap = Maps.Map_2

portalSceneObject = Portal(position, destinationMap)

scene.CreateSceneObject(portalSceneObject)

File: Maple Fighters/MainServers/Game/Game/Application/Game Objects/Portal/Portal.cs
using MathematicsHelper;
using Game.Common;
using InterestManagement;

namespace Game.Application.GameObjects
{
    public class Portal : GameObject
    {
        public Portal(Vector2 position, Maps destinationMap) 
            : base("Portal", new TransformDetails(position, Vector2.Zero, Direction.Left))
        {
            Components.AddComponent(new PortalInfoProvider(destinationMap));
        }
    }
}